deploy-agent/deployd/types/ping_request.py

```diff
@@ -71,9 +71,11 @@ def to_json(self):
             ping_report["errorMessage"] = report.errorMessage
             ping_report["failCount"] = report.failCount
             ping_report["deployAlias"] = report.deployAlias
+            """
```

Comment:
why this?

Reply:
If we use this, the server responds with "json error".

Reply:
since we didn't send any content for extraInfo before, I guess it is some code never tested. After I comment it out, the server responds 200 ok which means the map is accepted by java server.